src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="group">
+        <legend>Personal information </legend>
+        <label for="surname" class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            required
+            autocomplete="off"
+            maxlength="30"
+          >
+        </label>
+        <label for="name" class="form-field">
```

Comment:
Add empty lines between multiline siblings.
```suggestion
        </label>
        
        <label for="name" class="form-field">
```